Cargo.toml

```diff
@@ -59,7 +59,7 @@ home = "^0.5"
 indexmap = { version = "^1", features = ["serde-1"] }
 ignore = "^0.4"
 log = "^0.4"
-regex = "^1"
+regex = "1.5.5"
```

Comment:
i know its default in cargo, but its not the semver way. i'm not sure why cargo decided that by default ^ is implied even if not written, but i feels its not readable.
can we just add the ^ so it will be ^1.5.5